qml/StatisticsChartBox.qml

```diff
@@ -201,6 +209,28 @@ Rectangle {
                     onTriggered: {
                         running ? statisticsChartViewLoader.item.dynamicPause() : statisticsChartViewLoader.item.dynamicContinue()
                     }
+
+                }
+            }
+        }
+
+        QtDialogs.FileDialog {
+            id: csvDialog
+            title: "Choose a file name and path to save csv"
```

Comment:
```suggestion
            title: chartbox ? "Save " + chartTitle + " chart box data" :
                    "Save series " + customLegend.getLabel(seriesIndex) + " from " + chartTitle
```

Reply:
:+1:

Reply:
Title in FileDialog is not visible, dont know why.

Reply:
This is the name of the new window opened.

Reply:
You missed this